integration_test/dr-buckets.tf.template

```diff
@@ -1,3 +1,6 @@
+# This will get the DR AWS account ID.
+data "aws_caller_identity" "current_account" {}
```

Comment:
consider renaming this from `current_account` to `dr_account` to better denote the accounts being leveraged in the policy document.

Reply:
Renamed to `dr_account `